mmpose/visualization/local_visualizer.py

```diff
@@ -225,8 +225,8 @@ def _draw_instances_kpts(self,
             keypoints = instances.get('transformed_keypoints',
                                       instances.keypoints)
 
-            if 'scores' in instances and self.show_keypoint_weight:
-                scores = instances.scores
+            if 'keypoint_scores' in instances:
```

Comment:
Why is `self.show_keypoint_weight` omitted here?

Reply:
I think `keypoint_scores` shoud be irrevelant to `self.show_keypoint_weight`. 

A special case is that when I do not want to draw those keypoints with `keypoint_scores` lower than `kpt_score_thr`. If  `self.show_keypoint_weight = False`, and `keypoint_scores` is still present in instances, then although `keypoint_scores` is present in instances,  they are all forced to be assigned 1, and will be draw in the end.

Reply:
Thanks, it makes sense. 

But `self.show_keypoint_weight` controls whether to plot keypoints with different transparency depending on keypoint_scores. When `self.show_keypoint_weight==False`, all visible keypoints should be plotted with the same transparency. This argument seems not to work now. To make it work, we also need to modify [this line](https://github.com/open-mmlab/mmpose/blob/e4549b13ea89dde824d5e4f90a7ef84c0364174d/mmpose/visualization/local_visualizer.py#L262).

Reply:
How about this
```
if self.show_keypoint_weight:
    transparency = max(0, min(1, score[kid])) * self.alpha
```

Reply:
`transparency` needs to be set when `self.show_keypoint_weight` is False
```python
transparency = self.alpha
if self.show_keypoint_weight:
    transparency *= max(0, min(1, score[kid]))  
```

Reply:
Nice. I will fix it according to your advice.